msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/Authority.java

```diff
@@ -63,8 +62,10 @@ static Authority createAuthority(URL authorityUrl) throws MalformedURLException{
             createdAuthority = new B2CAuthority(authorityUrl);
         } else if (authorityType == AuthorityType.ADFS) {
             createdAuthority = new ADFSAuthority(authorityUrl);
-        } else if(authorityType == AuthorityType.CIAM){
```

Comment:
The fact that you modified this existing code and no unit test broke is an indication of a test gap. Please add a unit test to get coverage of this logic.

Reply:
This change was unnecessary and revert in the latest commit (more context in your other comment in this file: https://github.com/AzureAD/microsoft-authentication-library-for-java/pull/851#discussion_r1705299331)

We have existing unit tests covering some of these methods with various authority types (https://github.com/AzureAD/microsoft-authentication-library-for-java/blob/dev/msal4j-sdk/src/test/java/com/microsoft/aad/msal4j/AuthorityTest.java), however none of them broke because they all succeeded before they would have gotten to the new code.